### DISCUSSION TITLE
is there any mkdocs alternative to sphinx apidoc?

### DISCUSSION THREAD

**Discussion by vmgustavo** [mkdocs/mkdocs#2885] [Q&A]
Is there any method to automatically create the markdown files based on the python module structure similar to what [sphinx-apidoc](https://www.sphinx-doc.org/en/master/man/sphinx-apidoc.html) does?

**Comment by pawamoy** [comment]
[mkdocstrings](https://github.com/mkdocstrings/mkdocstrings)? Coupled with [this recipe](https://mkdocstrings.github.io/recipes/#automatic-code-reference-pages)? :slightly_smiling_face:

**Reply by vmgustavo** [reply]
thanks, I will take a look at it
